.gitlab-ci.yml

```diff
@@ -327,9 +329,13 @@ build_qemux86_64_uefi_grub:
     JOB_BASE_NAME: mender_qemux86_64_uefi_grub
     ONLY_BUILD: "true"
   only:
+    # The variables expressions will work after GitLab release 12.0,
+    # see https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/27925
+    # Meanwhile split them into two different conditions as a workaround
     variables:
       - $BUILD_QEMUX86_64_UEFI_GRUB == "true"
-      - $BUILD_VEXPRESS_QEMU == ""
+    variables:
+      - $RUN_INTEGRATION_TESTS == "true"
```

Comment:
I think this will not work. Separating the conditions in different variables tag is an `and`. So now they will only run when RUN_INTEGRATION_TESTS and BUILD_QEMUX86_64_UEFI_GRUB. For an `or` operation they should be in the same variables tag

Reply:
Ah, I thought it was the opposite! Will fix this.

Reply:
Your comments about "and/or" of variables made me realize my assumptions were wrong, and your original code was already correct. My mistake! I will remove this commit entirely.

Reply:
A somewhat related question though: We test for `TEST_VEXPRESS_QEMU == ""`, but in the variable list you have things like: `TEST_RASPBERRYPI3: "false"`. Will it be translated to an empty value, or is this comparison wrong?

Reply:
The logic was correct but still could have been simplified but not taking into account this strange corner case: willing to run integ tests with vxpress client.

Indeed, the comparison is wrong. Said in other words, it is just inconsistent, as the default values for GitLab is "false" string but the release_tool used to send empty values for off. Now that has been changed from the release_tool side (as it fetches this same defaults) so it could be cleaned up. Anyway, this is a corner case and not very important IMO.

Reply:
> The logic was correct but still could have been simplified but not taking into account this strange corner case: willing to run integ tests with vxpress client.

I suppose it might be time to retire vexpress-qemu integration testing. No currently supported Yocto branch (sumo, rocko, thud) is missing x86 support, so they will all be able to run them still. In that case the whole job can be removed. Do you agree?

Reply:
I think that is a great idea. But let's do it in a separate PR and take the opportunity to clean up the job (it will not need to be a template anymore and other details in the previous build stages can be cleaned).

I will do this :)

Reply:
Great, thanks!